mpu6050/mpu6050.sh

```diff
@@ -14,11 +9,18 @@ do
 	MAJOR=$(i2cget -y 0 0x68 0x41)
 	MINOR=$(i2cget -y 0 0x68 0x42)
 	
-	Temperature=$(( -(((MAJOR << 8)+ MINOR) ^ ((1 << 0x10) - 1)) ))
+	SIGNED=$(( (MAJOR & 0x80) >> 7))
+	Temperature=$(( (MAJOR << 8) + MINOR))
+	
+	if [ $SIGNED -eq 1 ] ; then
+		Temperature=$(( -(Temperature ^ 0xFFFF) ))
```

Comment:
This conversation is not correct. Absolute value of 2complemeted 16 bit integer is  (0x10000 - value).